era/threat/fight.js

```diff
@@ -141,7 +129,7 @@ export class Fight {
    * @returns {[Record<string, Unit>, Record<string, Unit>]}
    */
   eventToFriendliesAndEnemies(ev, unit) {
-    let friendly = ev[unit + "IsFriendly"];
+    let friendly = ev[unit + "IsFriendly"] ?? ev.type === "combatantinfo";
```

Comment:
The main change, `combatantinfo` event has a `e.sourceID`, but no `e.sourceIsFriendly`, and that caused some issues swapping friendlies and enemies. This is a simple fix for it.